bliss/encoder.py

```diff
@@ -93,6 +93,13 @@ def dist_param_groups(self):
         }
 
     def encode_batch(self, batch):
+        img_dims = list(batch["images"].size())
+        images_with_background = torch.zeros(
+            img_dims[0],
+            img_dims[1] * 2,
+            img_dims[2],
+            img_dims[3],
+        )
```

Comment:
Is this debugging code that got left in? (Why are we setting `images_with_background` here if we're just going to redefine it on right away?)

Reply:
Yes it was - I added the `torch.cat` line afterwards to fix the issue. I'll remove the first definition of `images_with_background`.